onnx/backend/test/case/node/optionalhaselement.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+from typing import Optional, List, Any
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+def optional_has_element_reference_implementation(optional):  # type: (Optional[np.ndarray]) -> np.ndarray
+    if optional is None:
+        return np.array(False)
```

Comment:
Does this map to a one-dimensional tensor in ONNX? I think we want a scalar (zero-dimensional tensor), right?

Reply:
This is a zero dimension numpy scalar. The ONNX model output value info does not have shape (only boolean dtype).